src/spec/page-objects/settings-page.js

```diff
@@ -57,5 +58,13 @@ class SettingsPage {
       return val[0].click();
     });
   }
+  getDisabledDomain(){

```

Comment:
And what is `['Activate']`? I mean, how do you name this method to be precise? maybe `getDisableDomainButtonTexts()`?

I think that maybe you can create a new method `getDisableButtonsCount()` in replace of this one, I think that it could be easy to understand.